main/lua-stdlib/APKBUILD

```diff
@@ -14,18 +14,12 @@ for _v in $_luaversions; do
 	subpackages="$subpackages lua$_v-stdlib:split_${_v/./_}"
 done
 source="$pkgname-$pkgver.tar.gz::https://github.com/lua-stdlib/lua-stdlib/archive/release-v$pkgver.tar.gz"
-_sdir="$srcdir"/lua-stdlib-release-v$pkgver
+builddir="$srcdir"/lua-stdlib-release-v$pkgver
 
 prepare() {
-	cd "$_sdir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
```

Comment:
:+1: for removing this, please call `default_prepare` from `prepare` though to ensure that patches added to `$source` (in the future) are actually applied.